### ISSUE TITLE
TOC orientation

### ISSUE THREAD

**Issue by dag7dev** [tomfran/typo#30]
Currently toc is at the beginning, it would be great if we're allowed to select where to show it (for example on the left or on the right side of the page).

**Comment by tomfran** [comment]
Yes it can be a viable option, but it requires some thinking, for instance, what would you expect when viewing on mobile? Also, I guess it would be cool to have it highlight the current "focused" section, something that now is not needed as the TOC gets basically out of the way as soon as you click it.

Feel free to tackle this yourself if you know how to do it, I honestly am not a huge expert on aligning stuff on the web, and I feel this would disrupts a lot of the current implementation.

**Comment by dag7dev** [comment]
on mobile it stays as it is now, I don't think there are other viable options. Maybe a good enhancement may it be a button to "go to the top of the page", but this is another matter.

**Comment by tomfran** [comment]
There is already a back to top button at the end of pages, perhaps you're referring to something appearing as you scroll, I guess yes it would be nice to have

**Comment by tomfran** [comment]
Closing as this is not planned.
